packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -361,6 +361,12 @@ module.exports = (
           })
         },
       },
+      rawMarkdownBody: {
+        type: GraphQLString,
+        resolve(markdownNode) {
+          return markdownNode.rawMarkdownBody
+        },
+      },
```

Comment:
is this needed? 

it should work without adding anything here (if `data.content` used for `rawMarkdownBody` in `on-node-create.js` is string)

Reply:
You're right. It's unnecessary. My bad. I'll make a new commit without it.